docs/configuration.asciidoc

```diff
@@ -170,7 +174,7 @@ Also keep that in mind when creating alerts.
 [[config-transaction-sample-rate]]
 ==== `transaction_sample_rate`
 
-By default, the agent will sample every transaction (e.g. request to your service). To reduce overhead and storage requirements, you can set the sample rate to a value between 0.0 and 1.0. We still record overall time and the result for unsampled transactions, but no context information, labels, or spans.
+By default, the agent will sample every transaction (e.g. request to your service). To reduce overhead and storage requirements, you can set the sample rate to a value between 0.0 and 1.0. We still record overall time and the result for unsampled transactions, but no context information, tags, or spans.
```

Comment:
I fixed this in my PR (ie change in `CoreConfiguration.java`), so no need to fix if you encounter this

Reply:
This was just a byproduct of executing the tests 🙂 (ConfigurationExporterTest)

Reply:
Yes, I meant I updated the origin (the java class) in my PR